frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/setup.tsx

```diff
@@ -23,6 +24,7 @@ export const setupEmbedding = async ({
   hasEnterprisePlugins = false,
 }: SetupOpts) => {
   const returnedValue = await setup({
+    settings: [createMockSettingDefinition({ key: "enable-embedding" })],
```

Comment:
because we change the way we define this setting, we change it from defining it in the `settings` to rendering the toggle inside a component instead.
https://github.com/metabase/metabase/blob/191d832d8fac25f1c0f510d0ad92b12cc2e460e6/frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSettings.tsx#L31-L44